### PR TITLE
Bug 1998394: add tests for RHEL9 template

### PR DESCRIPTION
**Pull request by gouyang**
review notes:
1. upstream has no RHEL9 template, move the test downstream only
2. there is a test failure on `action/clone.spec.ts`, try to fix the flaky here.
3. add a new constant `TEMPLATE`.

### CODE REVIEW

**Comment by gouyang** [comment]
@yaacov @pcbailey PTAL.

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts [hunk 1]
```diff
@@ -94,9 +90,61 @@ export enum DISK_DRIVE {
   CDROM = 'CD-ROM',
 }
 
-export enum OS {
-  fedora = 'Fedora 32 or higher',
-  win10 = 'Microsoft Windows 10',
-  win2k12 = 'Microsoft Windows Server 2012 R2',
-  rhel8 = 'Red Hat Enterprise Linux 8.0 or higher',
-}
+export const TEMPLATE = {
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts:93]: This looks so nice! =)

**Review by pcbailey — COMMENTED** [review]
/lgtm

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts (hunk 1 above)
Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts:93]: Thanks！

**Pull request merged**